source/docs/essentials/installation.md

```diff
@@ -1,6 +1,7 @@
 ---
 id: installation
 title: Installation
+description: Front-Commerce is a Node.js server that will serve a GraphQL endpoint and a React application to your customers. In order to run it, you need to make sure you have `npm` and `node` installed on your machine in a supported version.
```

Comment:
```suggestion
description: Front-Commerce is a Node.js server that will serve your customers a GraphQL endpoint and a React application. To run it, you need to make sure you have `npm` and `node` installed on your machine in a supported version.
```

Not sure which is better in terms of searchability

- `Front-Commerce is a Node.js server that will serve your customer ...`
- `Front-Commerce is a Node.js server that will serve a GraphQL endpoint and a React application ...`

Reply:
I think the tech part is most important here. I'll leave the initial version.